File: dotnet/source/cmf.bus/IEnvelopeBus.cs
namespace cmf.bus
{
    /// <summary>
    /// A convenience interface that combines the <see cref="IEnvelopeReceiver">IEnvelopeReceiver</see> 
    /// and <see cref="IEnvelopeSender">IEnvelopeSender</see> interfaces together because they are most 
    /// typically implemented and referenced together.
    /// </summary>
    public interface IEnvelopeBus : IEnvelopeSender, IEnvelopeReceiver
    {
    }
}

File: java/cmf-bus-api/src/main/java/cmf/bus/IEnvelopeBus.java
package cmf.bus;

/**
 * A convenience interface that combines the {@link IEnvelopeReceiver} and {@link IEnvelopeSender} 
 * interfaces together because they are most typically implemented and referenced together.
 */
public interface IEnvelopeBus extends IEnvelopeReceiver, IEnvelopeSender, IDisposable {

}
